config/routes.rb

```diff
@@ -74,6 +71,11 @@
   match '/manage/phone' => 'users/phones#update', via: [:patch, :put]
   get '/manage/recovery_code' => 'users/recovery_codes#show'
 
+  get '/otp/send' => 'users/two_factor_authentication#send_code'
+  get '/phone_setup' => 'users/two_factor_authentication_setup#index'
+  patch '/phone_setup' => 'users/two_factor_authentication_setup#set'
+  get '/users/two_factor_authentication' => 'users/two_factor_authentication#show', as: :user_two_factor_authentication
```

Comment:
I know you're not a fan of comments, but maybe this would be a good opportunity to comment this? 😬 

```
# route name is used by two_factor_authentication gem
```